src/est/src/EstimateParasitics.cpp

```diff
@@ -683,7 +683,7 @@ void EstimateParasitics::makePadParasitic(const Net* net,
     arc_delay_calc_->reduceParasitic(
         parasitic, net, corner, sta::MinMaxAll::all());
   }
-  parasitics_->deleteParasiticNetworks(net);
+  // parasitics_->deleteParasiticNetworks(net);
```

Comment:
Why is this removed?  Same below

Reply:
I suggested this change as an experiment to see if Cho's case of stale parasitics was related to what I was debugging. Ultimately I arrived at the fix https://github.com/The-OpenROAD-Project/OpenROAD/pull/8991